testpilot/frontend/static-src/images/share-email.svg

```diff
@@ -0,0 +1,26 @@
+                    <path d="M9.99678245,10.5 C9.4462747,10.5 9,10.9510941 9,11.5010545 L9,21.0060999 C9,21.5589671 9.40075099,22.2256356 9.8756078,22.4845177 L27.8902898,32.3057505 C29.0554506,32.9409721 30.943123,32.9417498 32.1097102,32.3057505 L50.1243922,22.4845177 C50.607977,22.2208773 51,21.5560603 51,21.0060999 L51,11.5010545 C51,10.9481874 50.5567382,10.5 50.0032175,10.5 L9.99678245,10.5 Z" id="Mask" fill="#F1F1F1"></path>
+                    <rect id="lines" fill="#E66000" x="12" y="13.5" width="12" height="2.25" rx="1"></rect>
+                    <rect id="lines" fill="#E66000" x="12" y="17.25" width="16.5" height="2.25" rx="1"></rect>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Not sure if we want to run these through [**svgo**](https://www.npmjs.com/package/svgo) (or if we do optimization somewhere else in the Node/Python pipeline), but it looks like we can shed about 59% of the filesize:

```sh
$ svgo -i share-email.svg -o share-email-optimized.svg

Done in 129 ms!
8.763 KiB - 59.5% = 3.552 KiB
```